packages-web/accessibility-helper/src/AccessibilityHelper.tsx

```diff
@@ -0,0 +1,91 @@
+import { createElement, MutableRefObject, ReactElement, useCallback, useEffect, useRef } from "react";
+
+import { ValueStatus, DynamicValue } from "mendix";
+
+import { AccessibilityHelperContainerProps, AttributesListType } from "../typings/AccessibilityHelperProps";
+
+const PROHIBITED_ATTRIBUTES = ["class", "style", "widgetid", "data-mendix-id"];
+
+const AccessibilityHelper = (props: AccessibilityHelperContainerProps): ReactElement => {
+    const contentRef: MutableRefObject<HTMLDivElement | null> = useRef(null);
+    let observer;
+
+    const update = useCallback(() => {
+        if (contentRef.current) {
+            try {
+                const target = contentRef.current.querySelector(props.targetSelector);
+                if (target) {
+                    props.attributesList.forEach(attrEntry => {
+                        const valueToBeSet = getValueBySourceType(attrEntry);
+                        const condition = attrEntry.attributeCondition;
+
+                        if (condition.status === ValueStatus.Available && condition.value) {
+                            if (
+                                valueToBeSet?.status === ValueStatus.Available &&
+                                valueToBeSet?.value !== target.getAttribute(attrEntry.attribute)
+                            ) {
+                                target.setAttribute(attrEntry.attribute, valueToBeSet?.value);
+                            }
+                        } else if (condition.status === ValueStatus.Available && !condition.value) {
+                            target.removeAttribute(attrEntry.attribute);
+                        }
+                    });
+                }
+            } catch (error) {
+                console.error(`Accessibility Helper selector ${props.targetSelector} is not valid`, error);
+            }
+        }
+    }, [props.targetSelector, props.attributesList]);
+
+    useEffect(() => {
+        if (!isValid(props.targetSelector)) {
+            return;
+        }
+        update();
+    }, [props.attributesList]);
```

Comment:
Forgot `props.targetSelector`

Reply:
`props.attributesList` is already being handled in the other useEffect

Reply:
Other use effect will also trigger on contentRef changes which we dont want